### PR TITLE
create transformation

### PR DESCRIPTION
**Pull request by ameydwivedi**
created the create transformation operation in azure stream analytics

### CODE REVIEW

File: cs-azure/src/main/java/io/cloudslang/content/azure/actions/streamanalytics/transformations/CreateTransformation.java [hunk 1]
```diff
@@ -0,0 +1,142 @@
+/*
```
Comment (manjoosh12) [inline cs-azure/src/main/java/io/cloudslang/content/azure/actions/streamanalytics/transformations/CreateTransformation.java:1]: copyrights has been added twice

File: cs-azure/src/main/java/io/cloudslang/content/azure/entities/CreateTransformationInputs.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
+package io.cloudslang.content.azure.entities;
```
Comment (manjoosh12) [inline cs-azure/src/main/java/io/cloudslang/content/azure/entities/CreateTransformationInputs.java:1]: please add the copy rights

File: cs-azure/src/main/java/io/cloudslang/content/azure/entities/models/streamanalytics/CreateTransformationRequestBody.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
+package io.cloudslang.content.azure.entities.models.streamanalytics;
```
Comment (manjoosh12) [inline cs-azure/src/main/java/io/cloudslang/content/azure/entities/models/streamanalytics/CreateTransformationRequestBody.java:1]: add the copy rights

File: cs-azure/src/main/java/io/cloudslang/content/azure/actions/streamanalytics/transformations/CreateTransformation.java (hunk 1 above)
Comment (ameydwivedi) [reply cs-azure/src/main/java/io/cloudslang/content/azure/actions/streamanalytics/transformations/CreateTransformation.java:1]: resolved

File: cs-azure/src/main/java/io/cloudslang/content/azure/entities/CreateTransformationInputs.java (hunk 2 above)
Comment (ameydwivedi) [reply cs-azure/src/main/java/io/cloudslang/content/azure/entities/CreateTransformationInputs.java:15]: resolved

File: cs-azure/src/main/java/io/cloudslang/content/azure/entities/models/streamanalytics/CreateTransformationRequestBody.java (hunk 3 above)
Comment (ameydwivedi) [reply cs-azure/src/main/java/io/cloudslang/content/azure/entities/models/streamanalytics/CreateTransformationRequestBody.java:15]: resolved

Comment (ameydwivedi) [reply cs-azure/src/main/java/io/cloudslang/content/azure/entities/models/streamanalytics/CreateTransformationRequestBody.java:15]: resolved

**Review by KalyankarRajesh — APPROVED** [review]

**Review by sushilMF — APPROVED** [review]

**Pull request merged**